helm/object-storage-operator/templates/deployment.yaml

```diff
@@ -43,7 +43,7 @@ spec:
         livenessProbe:
           httpGet:
             path: /healthz
-            port: {{ default 4444 .Values.listenPort }}
+            port: {{ trimPrefix ":" .Values.probeAddr | default 14001 }}
```

Comment:
Also, you have to add them here https://github.com/giantswarm/object-storage-operator/blob/98cce845b4e099bc6695be3aab0ba9044d34af14/helm/object-storage-operator/templates/deployment.yaml#L36

Reply:
I'd prefer to have same behavior on CAPZ and others